app/src/main/res/layout/activity_upload_categories.xml

```diff
@@ -38,7 +38,9 @@
         android:textSize="@dimen/subtitle_text"
         android:text="@string/upload_flow_all_images_in_set"
         android:layout_below="@+id/categories_title"
-        tools:text="(For all images in set)" />
+        android:visibility="gone"
+        tools:text="(For all images in set)"
+        tools:visibility="gone"/>
```

Comment:
This line is can be removed (use `android:visibility` over `tools:visibility` (same in activity_upload_license)

Reply:
OK. Will remove these two lines.